math/max/bitwisemax.go

```diff
@@ -7,6 +7,7 @@
 
 package max
 
+// BitwiseMax This function returns the maximum integer using bit operations
 func BitwiseMax(a int, b int, base int) int {
```

Comment:
`Bitwise` - max is in the name of package.

```suggestion
// Bitwise computes using bitwise operator the maximum of all the integer input and returns it
func Bitwise(a int, b int, base int) int {
```

Reply:
Done